packages/app-extrinsics/src/Params/Account.tsx

```diff
@@ -28,12 +28,12 @@ export default class Account extends React.PureComponent<Props> {
     );
   }
 
-  onChange = (publicKey?: Uint8Array): void => {
+  onChange = (accountId?: string): void => {
     const { onChange } = this.props;
 
     onChange && onChange({
-      isValid: !!publicKey && publicKey.length === 32,
-      value: publicKey
+      isValid: !!accountId && accountId.length === 48,
```

Comment:
Generally ok, however there are some tricks. (I commented on the other PR as well). Polkadot also has the notion of "short addresses", i.e. 1 bytes, 2 byte, 4 byte and 8 byte. Additionally the address contains 2 checksums, so just checking the length misses that.

The best route of attack to check for validity (not the fastest, but safest) is to -

1. Take address, decode it
2. If it throws, not valid
3. Check length (decoded publicKey) for a length of 32 bytes

For now, the decoding does not yet cater for the short addresses, so the first part of the comment is not catered for byt this. However, as soon as the decoder does, we can adapt here. In the meantime, we ensure that it is an actual real address and not just 48 random characters.